SECURITY.md

```diff
@@ -2,58 +2,41 @@
-and that we minimize the chance of critical bugs as much as possible.
+## Known Vyper Vulnerabilities
 
-Here are the audits we have undergone in the past:
-
-| Audit Type | Audit Date | Auditor | Version | Report Link |
-| ---------- | ---------- | ------- | ------- | ----------- |
-| Preliminary Review | October 28, 2019 | [ConsenSys Diligence](https://consensys.net/diligence/) | 0.1.0b13 | https://consensys.net/diligence/audits/2019/10/vyper/ |
-
-### Major Project Audits
-
-Please read prior audit reports for projects that use Vyper here:
-
-<!-- Please use the tagged version if possible, or commit hash if a non-tagged version was used. -->
-
-| Project | Version | Report Link |
-| ------- | ------- | ----------- |
-| [Uniswap](https://uniswap.io) | 35038d2 | https://medium.com/consensys-diligence/uniswap-audit-b90335ac007 |
-| [Computable](https://github.com/computablelabs/computable) | 0.1.0b10 | https://github.com/trailofbits/publications/raw/master/reviews/computable.pdf |
-
-## Known Vyper Vulnerabilities and Exposures (VVEs)
-
-The link below is a list of all publicly disclosed vulnerabilities and exposures.
+- The link below is a list of all publicly disclosed vulnerabilities and exposures.
 Best Practices dictate that when we are first made aware of a potential vulnerability,
-we take the precaution of assessing it's potential impact to deployed projects first.
+we take the precaution of assessing its potential impact to deployed projects first.
 When we are confident that a disclosure will not impact known projects that use Vyper,
 we will add an entry to the list of security advisories for posterity and reference by others.
 
-https://github.com/vyperlang/vyper/security/advisories
+  - list of publicly known vulnerabilities: https://github.com/vyperlang/vyper/security/advisories
+
+
+## Bug Bounty Program
+- as of May 2024, Vyper does not have a bug bounty program. It is planned to instantiate one in the near future.
 
 ## Reporting a Vulnerability
 
-If you think you have found a security vulnerability with a project that has used Vyper,
+If you think you have found a security vulnerability caused by the compiler with a project that has used Vyper,
 please report the vulnerability to the relevant project's security disclosure program prior
-to reporting to us. If one is not available, submit it at https://github.com/vyperlang/vyper/security/advisories.
+to reporting to us. Additionally, please submit the compiler vulnerability at https://github.com/vyperlang/vyper/security/advisories.
 
 **Please Do Not Log An Issue** mentioning the vulnerability.
 
-If you have contacted the relevant project, or you have found something that you do not think affects
-a particular project, please also email your vulnerability to security@vyperlang.org. Our PGP key is:
+
+The Vyper team can also be reached via email at security@vyperlang.org. Our PGP key is:
```

Comment:
i thought i removed this before. the email is not really monitored anymore

Reply:
we should still have an email for point of contact.

Reply:
we should still have an email for point of contact.

Reply:
> we should still have an email for point of contact.

why?

Reply:
as a backup if the repo is taken down by GH but the security policy is still available in the internet archive for example

Reply:
Ok another idea: can we add somewhere `security.txt` and `/.well-known/security.txt` in the Vyper domain (see https://en.wikipedia.org/wiki/Security.txt), with the contact details you want.